src/js/stores/MetronomeStore.js

```diff
@@ -166,10 +166,40 @@ class MetronomeStore extends EventEmitter {
     MetronomeActions.runJob(jobID);
   }
 
-  suspendSchedule(jobID) {
-    let schedule = this.getJob(jobID).schedules[0];
-    schedule.enabled = false;
-    MetronomeActions.suspendSchedule(jobID, schedule);
+  disableSchedule(jobID) {
+    let job = this.getJob(jobID);
+    if (job == null) {
+
+      return null;
+    }
+
+    let [schedule] = job.getSchedules();
+    if (schedule == null) {
+
+      return null;
+    }
+
+    this.updateSchedule(jobID, Object.assign({}, schedule, {enabled:false}));
+  }
+
+  enableSchedule(jobID) {
+    let job = this.getJob(jobID);
+    if (job == null) {
+
+      return null;
+    }
+
+    let [schedule] = job.getSchedules();
+    if (schedule == null) {
+
+      return null;
+    }
+
+    this.updateSchedule(jobID, Object.assign({}, schedule, {enabled:true}));
+  }
+
+  updateSchedule(jobID, schedule) {
```

Comment:
You mean this https://github.com/dcos/dcos-ui/pull/639/files/8d027183f2820e3ae1afbbfaf31043feff55938c#diff-cad4282743a1728a5757cf28370730acR193 check?